### PR TITLE
Updates User's Availability when they move to a different week.

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]

File: src/main/webapp/availability.js [hunk 1]
```diff
@@ -59,7 +59,11 @@ function updateAvailability() {
   };
   let requestBody = JSON.stringify(requestObject);
   let request = new Request('/availability', {method: 'PUT', body: requestBody});
-  fetch(request).then(() => location.reload()).catch((error) => {
+  fetch(request).then(() => {
+      if (buttonType === 'updateButton') {
```
Comment (apluscs) [inline src/main/webapp/availability.js]: Some docu explaining why this check would be good

**Review by apluscs — APPROVED** [review]

File: src/main/webapp/availability.js (hunk 1 above)
Comment (meagancheese) [reply src/main/webapp/availability.js]: Done

File: src/main/webapp/availability.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ function availabilityTableDiv() {
   return document.getElementById('table-container');
 }
 
-function updateAvailability() {
+function updateAvailability(buttonType) {
```
Comment (jalexanderqed) [inline src/main/webapp/availability.js]: Change this to a boolean indicating whether to reload the page would probably make this interface simpler.

**Review by jalexanderqed — CHANGES_REQUESTED** [review]

File: src/main/webapp/availability.js (hunk 2 above)
Comment (meagancheese) [reply src/main/webapp/availability.js]: Done

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**